File: HerenciaCSharp/HerenciaCSharp/Estudiante.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace HerenciaCSharp
{
    public class Estudiante : Persona
    {
        private String Carrera;

        public void setCarrera(String val)
        {
            Carrera = val;
        }
        public String getCarrera()
        {
            return Carrera;
        }
    }
}

File: HerenciaVisual/HerenciaVisual/Estudiante.vb
Public Class Estudiante

    Inherits Persona
    Private carrera As String

    Sub New()
    End Sub

    Sub New(ByVal carrera1 As String)
        Me.carrera = carrera1
    End Sub

    Public Sub SetCarrera(ByVal carrera1 As String)
        carrera = carrera1
    End Sub

    Public Function GetCarrera() As String
        Return carrera
    End Function

    Overrides Sub Imprimir()
        MessageBox.Show(MyBase.GetNombre)
        MessageBox.Show(MyBase.GetTelefono)
        MessageBox.Show(Me.GetCarrera)
        MessageBox.Show(Me.carrera)
    End Sub
End Class
